hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/audit/OMSystemAction.java

```diff
@@ -22,7 +22,12 @@
  * as present for request.
  */
 public enum OMSystemAction implements AuditAction {
-  STARTUP;
+  STARTUP,
+
+  //Background Deletion Actions
+  DIRECTORY_DELETION,
+  KEY_DELETION,
+  SNAPSHOT_DELETION;
```

Comment:
@sarvekshayr , It would also be beneficial to use more operation types to track the full path of snapshot deletion SNAPSHOT_DELETION, SNAPSHOT_MOVE_DEL_KEYS, SNAPSHOT_PURGE.
include the snapshot ID if the Key/Directory deletion was performed as part of snapshot cleanup.